NextcloudTalk/NCAPIController.m

```diff
@@ -324,8 +324,7 @@ - (NSURLSessionDataTask *)getContactsForAccount:(TalkAccount *)account forRoom:(
         NSMutableArray *users = [[NSMutableArray alloc] initWithCapacity:responseContacts.count];
         for (NSDictionary *user in responseContacts) {
             NCUser *ncUser = [NCUser userWithDictionary:user];
-            TalkAccount *activeAccount = [[NCDatabaseManager sharedInstance] activeAccount];
-            if (ncUser && !([ncUser.userId isEqualToString:activeAccount.userId] && [ncUser.source isEqualToString:kParticipantTypeUser])) {
+            if (ncUser && ([ncUser.userId isEqualToString:account.userId] && [ncUser.source isEqualToString:kParticipantTypeUser])) {
```

Comment:
Hm... This changes the condition?

Reply:
Oopsie, good catch! push that change by mistake 😅